### ISSUE TITLE
"pip install blobxfer --upgrade" failing on CentOS 7.2

### ISSUE THREAD

**Issue by xpillons** [Azure/blobxfer#14]
While executing "pip install blobxfer -upgrade" on CentOS 7.2, the command is failing with the following error 
    build/temp.linux-x86_64-2.7/_openssl.c:12:24: fatal error: pyconfig.h: No such file or directory
     #  include <pyconfig.h>

prior to run this command I ran
yum -y install python3-pip libssl-dev libffi-dev npm
pip install --upgrade pip

can you please share the way to setup blobxfer on CentOS ?

**Comment by alfpark** [comment]
@xpillons Try the following (this installs for Python 2, which is available out of the box on CentOS 7.2):

```shell
sudo yum install -y gcc openssl-devel libffi-devel python-devel
curl https://bootstrap.pypa.io/get-pip.py | sudo python
sudo pip install --upgrade blobxfer
```

**Comment by xpillons** [comment]
Works. Thanks.
